server/services/DashboardService/index.js

```diff
@@ -0,0 +1,74 @@
+import { createHash } from 'crypto'
+import _ from 'lodash'
+import CollectionsModel from '../../models/CollectionsModel'
+import MetadataModel from '../../models/MetadataModel'
+import DashboardDataProcessor from '../../data_processors/DashboardDataProcessor'
+
+class DashboardService {
+  #assessmentKey = 'analysis'
```

Comment:
just checking - are we able to support the new `#` private attribute feature in all environments? I don't think it's fully released yet.

Reply:
Support looks pretty good
<img width="798" alt="Screen Shot 2023-07-28 at 2 05 47 PM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/e67d22a9-aebc-4c8f-aace-1ccf52e215a8">